src/Sentry/SentryEvent.cs

```diff
@@ -242,6 +248,16 @@ public void WriteTo(Utf8JsonWriter writer, IDiagnosticLogger? logger)
             writer.WriteDictionaryIfNotEmpty("extra", _extra, logger);
             writer.WriteStringDictionaryIfNotEmpty("tags", _tags!);
 
+            if (SentryDebugImages != null && SentryDebugImages.Count > 0)
```

Comment:
Does it work?

```suggestion
            if (SentryDebugImages?.Count > 0)
```

Reply:
> For the [comparison operators](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/comparison-operators) <, >, <=, and >=, if one or both operands are null, the result is false.

That is good to know, I wasn’t sure about the nullability rules in C#.